.github/workflows/sanity.yaml

```diff
@@ -21,7 +21,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.17'
+      - id: golangci-lint
+        run: echo "::set-output name=version::$(cd hack/tools; go list -m github.com/golangci/golangci-lint | cut -d' ' -f2)"
       - name: Run linting checks
-        uses: "golangci/golangci-lint-action@v2"
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: ${{ steps.golangci-lint.outputs.version }}
```

Comment:
Rather than hardcoding to 1.43 here (and risking getting out of sync with what's in our `hack/tools/go.mod`), I thought "let's just read our go.mod and use that!"

Reply:
Do we need a `needs` keyword here if we're reliant on the output of parsing the go.mod in the previous job step?